esphome/components/web_server_idf/web_server_idf.cpp

```diff
@@ -62,20 +63,62 @@ void AsyncWebServer::begin() {
   }
 }
 
+esp_err_t AsyncWebServer::request_post_handler(httpd_req_t *r) {
+  ESP_LOGVV(TAG, "Enter AsyncWebServer::request_post_handler. uri=%s", r->uri);
+  auto content_type = request_get_header(r, "Conetnt-Type");
```

Comment:
Not sure how this was working in your testing...
```suggestion
  auto content_type = request_get_header(r, "Content-Type");
```

Reply:
> Not sure how this was working in your testing...

This check is only to forbid multipart requests. My tests using only form-urlencoded request, so they works. Thanks for the typo found!